examples/todo-typescript/src/client/MainPage.tsx

```diff
@@ -1,17 +1,11 @@
+import { getUsername, type AuthUser as User } from "wasp/auth";
```

Comment:
Maybe keep `AuthUser` as symbol name instead of aliasing it as `User`?

Reply:
Will do